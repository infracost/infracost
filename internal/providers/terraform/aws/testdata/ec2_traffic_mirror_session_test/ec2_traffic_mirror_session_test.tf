provider "aws" {
  region                      = "us-east-1"
  skip_credentials_validation = true
  skip_metadata_api_check     = true
  skip_requesting_account_id  = true
  skip_region_validation      = true
  access_key                  = "mock_access_key"
  secret_key                  = "mock_secret_key"
}

resource "aws_ec2_traffic_mirror_session" "session" {
  description              = "traffic mirror session"
  network_interface_id     = "eni-1234567"
  traffic_mirror_filter_id = "a-traffic-filter-id"
  traffic_mirror_target_id = "a-traffic-target-id"
  session_number           = "1"
}
