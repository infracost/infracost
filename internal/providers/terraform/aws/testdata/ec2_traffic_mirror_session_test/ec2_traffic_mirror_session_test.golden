
 Name                                    Monthly Qty  Unit   Monthly Cost 
                                                                          
 aws_ec2_traffic_mirror_session.session                                   
 └─ Traffic mirror                               730  hours        $10.95 
                                                                          
 OVERALL TOTAL                                                     $10.95 
──────────────────────────────────
1 cloud resource was detected:
∙ 1 was estimated